jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/impl/ExasolSqlDialectIT.java

```diff
@@ -294,6 +294,65 @@ public void testErrorMessages() throws SQLException, FileNotFoundException {
         false, getConfig().debugAddress(), "", null);
     }
 
+    @Test
+    public void testVirtualSchemaImportFromJDBCWithConnectionName() throws SQLException, FileNotFoundException {
+        final String connectionString = "jdbc:exa:localhost:" + getPortOfConnectedDatabase();
+        String createConnection = "create connection VS_JDBC_WITH_CONNNAME_CONNECTION to '" + connectionString + "' user '"
```

Comment:
Add `final`.

Since there are many other places here where `final` is missing, I am not going to reiterate that for each. Please set up your IDE to automatically add `final`.

Reply:
done